nyc_taxi/dashboard.ipynb

```diff
@@ -11,12 +11,12 @@
+    "<img align='right' style='padding:0px 0px 0px 0px;' \n",
+    "src='./thumbnails/index.png'/>\n",
```

Comment:
I think this could be better displayed on the website, at the moment it makes the first two code cells quite small. Could you instead display the screenshot just below the first paragraph? Perhaps build the website locally for iterating on this change.

![image](https://github.com/holoviz-topics/examples/assets/35924738/5bbfa987-877b-420d-b9a7-0939f1ee48f7)

Reply:
Yes I also tried, and it's more complicated than it should be!

It turns out our Sphinx `conf.py` adds the `html_image` myst extension (https://myst-parser.readthedocs.io/en/latest/syntax/optional.html#html-images) that is used to let Sphinx know it should gather all the images it finds in `img` HTML tags.

> Allowed attributes are equivalent to the image directive: src, alt, class, width, height and name. Any other attributes will be dropped.

So the extension drops `styles` :( An alternative would be to use the myst syntax for images (https://myst-parser.readthedocs.io/en/latest/syntax/images_and_figures.html), but using that syntax means that it won't be displayed to users opening the notebook without the jupyterlab myst extension installed. Maybe, in the future, we'll add this extension to each project, but that will require more thought, so let's not do it now.

So my recommendation is to add this after the first paragraph, saving the image I shared below in `nyc_taxi/assets/dashboard.png`.

 ```
<img src='./assets/dashboard.png' alt="Panel Dashboard" align="center">
```

![dashboard](https://github.com/holoviz-topics/examples/assets/35924738/91346329-09c2-4752-9b5a-538a63e2a27d)